bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -148,8 +148,8 @@ public enum Filter implements DisplayableSpType, SequenceableSpType, LoggableSpT
         K_SHORT("K-short (2.15 um)", "K-short", new Some<>(2.15), Ictd.track("K-short")),
         K_BLUE("K-blue (2.06 um)", "K-blue", new Some<>(2.06),    Ictd.track("K-blue")),
         K_RED("K-red (2.31 um)", "K-red", new Some<>(2.31),       Ictd.track("K-red")),
-        JH("JH (spectroscopic)", "JH", new Some<>(1.39),          Ictd.track("JH")),
-        HK("HK (spectroscopic)", "HK", new Some<>(1.871),         Ictd.track("HK")),
+        JH("JH (spectroscopic)", "JH", new Some<>(1.3385),        Ictd.track("JH")),
```

Comment:
@jluhrs Do you think this affects the seqexec. I checked the code and I don't see the wavelength being used but maybe I'm missing something

Reply:
At any rate, this changes the `instrument:observingWavelength` field in the sequence and there seems to be no other related value in the sequence.

Reply:
Ok, we read it from that field then